plugin/db/mongodb/migrate.go

```diff
@@ -0,0 +1,95 @@
+package mongodb
+
+import (
+	"context"
+
+	// embed will embeds the migration schema.
+	_ "embed"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.uber.org/zap"
+
+	"github.com/pkg/errors"
+
+	"github.com/bytebase/bytebase/common/log"
+	"github.com/bytebase/bytebase/plugin/db"
+)
+
+const (
+	// migrationHistoryDefaultDatabase is the default database name for migration history.
+	migrationHistoryDefaultDatabase = "bytebase"
+	// migrationHistoryDefaultCollection is the default collection name for migration history.
+	migrationHistoryDefaultCollection = "migrationHistory"
```

Comment:
Is it possible to use "migration_history" to be consistent with other database engines?

Reply:
Sure.